src/helpers/helpers.core.js

```diff
@@ -306,13 +306,6 @@ var helpers = {
 
 		ChartElement.__super__ = me.prototype;
 		return ChartElement;
-	},
-
-	_deprecated: function(scope, value, previous, current) {
```

Comment:
It may be worth keeping this around as we may need it in future version

Reply:
Yeah, I had debated. I'll add it back